app/styles/_variables.scss

```diff
@@ -7,6 +7,7 @@ $button-background-color: #0095dd;
 $button-background-disabled-color: #8a9ba8;
 $button-background-hover-color: #08c;
 $error-background-color: #d63920;
+$warning-background-color: #FF9216;
```

Comment:
Would save me a ton of work if this got merged before the sass-variables PR. No kidding.